gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannelTest.java

```diff
@@ -303,6 +303,44 @@ public void footerPrefetch_reused() throws IOException {
     assertThat(rangeHeaders).containsExactly("bytes=8-9", "bytes=7-7").inOrder();
   }
 
+  @Test
+  public void read_onlyRequestedRange() throws IOException {
```

Comment:
this will test StorageClientReadChannel as well?

Reply:
yes.